presto-geospatial/src/main/java/io/prestosql/plugin/geospatial/GeoFunctions.java

```diff
@@ -26,7 +26,6 @@
 import com.esri.core.geometry.Point;
```

Comment:
There are some other callsites that use spatial reference that should be removed (e.g., stCentroid). Also, in all places that verify whether spatial references are similar, we should be checking that they are null instead. We only really support euclidean geometries at this point. Otherwise, spatial functions might return unexpected results depending on where the geometries originated.